service/controller/chart/resource/release/desired.go

```diff
@@ -46,7 +46,9 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
 			return nil, microerror.Mask(err)
 		}
 
-		valuesMD5Checksum = fmt.Sprintf("%x", md5.Sum(valuesYAML))
+		// MD5 is only used for comparison but we need to turn off gosec or
+		// linting errors will occur.
+		valuesMD5Checksum = fmt.Sprintf("%x", md5.Sum(valuesYAML)) // #nosec
```

Comment:
We use this MD5 checksum annotation to check if the helm release values have changed. This is because we can't rely on Helm to return the same values we provide.

I think MD5 is safe to use for this but FYI @stone-z 

```
apiVersion: application.giantswarm.io/v1alpha1
kind: Chart
metadata:
  annotations:
    chart-operator.giantswarm.io/values-md5-checksum: 124a7cadff8175fd68d0f8933b7fb161
```

Reply:
If it's for change detection and not protecting data, you're good to go :+1: